platform/server/elasticsearch/__tests__/call_api.test.ts

```diff
@@ -0,0 +1,76 @@
+import { Client } from 'elasticsearch';
+import { callAPI } from '../call_api';
+
+test('should call the api when it exists, with the right context and params', async () => {
+  let apiContext;
+  const baz = jest.fn(function() {
+    apiContext = this;
+  });
+  const client: any = { foo: { bar: { baz: baz }}};
+  const result = await callAPI(client, 'foo.bar.baz', {}, {});
```

Comment:
Doesn't look like we test the `result` in any of these tests. We need to `return` something from `baz` above and make sure we receive that here.